examples/textual_inversion/textual_inversion.py

```diff
@@ -569,17 +569,21 @@ def main():
 
     # Create the pipeline using using the trained modules and save it.
     if accelerator.is_main_process:
-        pipeline = StableDiffusionPipeline(
-            text_encoder=accelerator.unwrap_model(text_encoder),
-            vae=vae,
-            unet=unet,
-            tokenizer=tokenizer,
-            scheduler=PNDMScheduler.from_config(args.pretrained_model_name_or_path, subfolder="scheduler"),
-            safety_checker=StableDiffusionSafetyChecker.from_pretrained("CompVis/stable-diffusion-safety-checker"),
-            feature_extractor=CLIPFeatureExtractor.from_pretrained("openai/clip-vit-base-patch32"),
-        )
-        pipeline.save_pretrained(args.output_dir)
-        # Also save the newly trained embeddings
+        if args.push_to_hub and args.only_save_embeds:
+            logger.warn("Enabling full model saving because --push_to_hub=True was specified.")
+            only_save_embeds = False
```

Comment:
this is never used no? Is something missing here?

Reply:
Right, that was when I last merged the upstream and found that `only_save_embeds` and `args.only_save_embeds` seemed to be redundant.

I thought it was a leftover from when args.save_full_model was used, but obviously this was the reason to have it two times. Is it too unclean to set `args.only_save_embeds = False` there? It would avoid the duplicate variable, but on the other hand afterward `args` do not longer represent the original arguments.